CMakeLists.txt

```diff
@@ -19,6 +19,6 @@ if("${BUILD_TESTING}")
     message("Building tests")
 
     include(CTest)
-    add_subdirectory("tests")
+    add_subdirectory("tests/")
```

Comment:
For my own edification, is the `/` here necessary, or is it just to make things more explicit?

Reply:
TBH I also don't know about this particular case, but I'm pretty sure it matters in other scenarios. IIRC, asking CMake to install `"x"` into a directory `y` will result in `y/x`, but asking it to install `"x/"` into a directory `y` will result in `y/contents_of_x`, *i.e.* the slash determines whether or not the directory and the contents are copied to `y` or if just the contents are copied.